packages/open-vm-tools/open-vm-tools.spec

```
@@ -0,0 +1,106 @@
+%global buildver 17337674
+
+Name: %{_cross_os}open-vm-tools
+Version: 11.2.5
+Release: 1%{?dist}
+Summary: Tools for VMWare
```

```suggestion
Summary: Tools for VMware
```
(and in summary below, and in service unit)